tests/cupy_tests/core_tests/test_dlpack.py

```diff
@@ -38,25 +52,26 @@ def test_conversion(self, dtype):
         testing.assert_array_equal(orig_array.data.ptr, out_array.data.ptr)
 
 
-@testing.parameterize(*testing.product({
-    'memory': ('device', 'managed')
-}))
-class TestNewDLPackConversion(unittest.TestCase):
-
-    old_pool = None
-    new_pool = None
+class TestNewDLPackConversion:
 
-    def setUp(self):
+    @pytest.fixture(autouse=True, params=["device", "managed"])
```

Comment:
Q: Should this fixture also have a class scope?

Reply:
I'll try.  This was now only active for the tests in the class, I think.  But making it class scope didn't work, because it didn't parametrize the tests.
(I.e. I think this was right and did the right thing; but it's the first time I am putting fixtures into a class :))

Reply:
I see, I saw you had `scope='class'` in another test class, and was assuming the semantics is "the class-scoped fixture's lifetime spans over each class test method's invocation", but maybe I am mistaken.

Reply:
OK, will not try it, here is the explenation that I hope is correct:
The scope indicates on which level the fixture is applied.  `scope="class"` means we run the fixture only once per class.

Now, that works in the second test case, but I think it is actually less clear, so rather than adding it here, I'll remove it there.